include/ofi_net.h

```diff
@@ -101,6 +105,11 @@ static inline uint64_t ntohll(uint64_t x) { return x; }
 #endif
 
 
+enum {
+	FI_EINPROGRESS_ASYNC = 512,	/* Async sockets */
+	FI_EINPROGRESS_URING = 513,	/* io_uring */
```

Comment:
Internal definitions use the OFI prefix, so we reserve FI for the external API only.  (Yes, there are still some places where we haven't renamed things from fi to ofi, but that's the direction).

Reply:
I changed the definitions to use the OFI prefix. Thanks!